code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -38,43 +42,119 @@
+	path = /obj/item/clothing/suit/storage/flannel
+
+/datum/gear/suit/flannel_shirt/New()
+	..()
+	var/flannels = list()
+	flannels["black flannel shirt"] = /obj/item/clothing/suit/storage/flannel
+	flannels["grey flannel shirt"] = /obj/item/clothing/suit/storage/flannel/grey
+	flannels["red flannel shirt"] = /obj/item/clothing/suit/storage/flannel/red
+	flannels["brown flannel shirt"] = /obj/item/clothing/suit/storage/flannel/brown
+	flannels["yellow flannel shirt"] = /obj/item/clothing/suit/storage/flannel/yellow
+	flannels["green flannel shirt"] = /obj/item/clothing/suit/storage/flannel/green
+	flannels["yellow flannel shirt"] = /obj/item/clothing/suit/storage/flannel/yellow
+	flannels["aqua flannel shirt"] = /obj/item/clothing/suit/storage/flannel/aqua
+	flannels["blue flannel shirt"] = /obj/item/clothing/suit/storage/flannel/blue
+	flannels["purple flannel shirt"] = /obj/item/clothing/suit/storage/flannel/purple
+	gear_tweaks += new/datum/gear_tweak/path(flannels)
+
 /datum/gear/suit/hazard_vest
-	display_name = "hazard vest selection"
+	display_name = "hazard vest (selection)"
 	path = /obj/item/clothing/suit/storage/hazardvest
 
 /datum/gear/suit/hazard_vest/New()
 	..()
 	var/hazard = list()
-	hazard["hazard vest, orange"] = /obj/item/clothing/suit/storage/hazardvest
-	hazard["hazard vest, blue"] = /obj/item/clothing/suit/storage/hazardvest/blue
-	hazard["hazard vest, white"] = /obj/item/clothing/suit/storage/hazardvest/white
-	hazard["hazard vest, green"] = /obj/item/clothing/suit/storage/hazardvest/green
+	hazard["orange hazard vest"] = /obj/item/clothing/suit/storage/hazardvest
+	hazard["blue hazard vest"] = /obj/item/clothing/suit/storage/hazardvest/blue
+	hazard["white hazard vest"] = /obj/item/clothing/suit/storage/hazardvest/white
+	hazard["green hazard vest"] = /obj/item/clothing/suit/storage/hazardvest/green
 	gear_tweaks += new/datum/gear_tweak/path(hazard)
 
 /datum/gear/suit/hoodie
-	display_name = "hoodie"
+	display_name = "hoodie (selection, colourable)"
 	path = /obj/item/clothing/suit/storage/hooded/wintercoat/hoodie
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
+/datum/gear/suit/hoodie/New()
+	..()
+	var/l_hoodie = list()
+	l_hoodie["hoodie"] = /obj/item/clothing/suit/storage/hooded/wintercoat/hoodie
+	l_hoodie["short-sleeved hoodie"] = /obj/item/clothing/suit/storage/hooded/wintercoat/hoodie/short
+	l_hoodie["crop top hoodie"] = /obj/item/clothing/suit/storage/hooded/wintercoat/hoodie/crop
+	l_hoodie["thick hoodie"] = /obj/item/clothing/suit/storage/toggle/hoodie
+	gear_tweaks += new/datum/gear_tweak/path(l_hoodie)
+
+/datum/gear/suit/hoodie_altmisc
+	display_name = "hoodie, misc (selection)"
+	path = /obj/item/clothing/suit/storage/toggle/hoodie/cti
+
+/datum/gear/suit/hoodie_altmisc/New()
+	..()
+	var/hoodie_misc = list()
+	hoodie_misc["CTI hoodie"] = /obj/item/clothing/suit/storage/toggle/hoodie/cti
+	hoodie_misc["MU hoodie"] = /obj/item/clothing/suit/storage/toggle/hoodie/mu
```

Comment:
What is MU? If it is not in lore, remove it, please.

Reply:
It stands for "Mars University".

Reply:
Add the Mars University part to the item's description then, please.

Reply:
I'll rebrand CTI and NRTI hoodies and lunchboxes to existing universities in Auroran lore.